src/Components/Filters/Search.styles.scss

```diff
@@ -6,7 +6,7 @@
   display: flex;
   flex-wrap: wrap;
   align-items: stretch;
-  width: 100%;
+  width: 75%;
```

Comment:
space between search input and other filtres looks odd, if we want to reduce size of input box we may reduce `col-size`.

<img width="1072" alt="Screen Shot 2020-02-21 at 3 58 16 PM" src="https://user-images.githubusercontent.com/39295560/75026464-07890e80-54c3-11ea-890b-8c04302f19e6.png">